components/actiongroup/stories/actiongroup.stories.js

```diff
@@ -65,27 +85,145 @@ export default {
+HorizontalCompact.tags = ["!dev"];
+HorizontalCompact.args = {
+	compact: true
+};
+
+/**
+ * The vertical option should be reserved for when horizontal space is limited.
+*/
+
+export const Vertical = TreatmentTemplate.bind({});
+Vertical.tags = ["!dev"];
+Vertical.args = {
+	vertical: true
+};
+
+export const VerticalCompact = TreatmentTemplate.bind({});
+VerticalCompact.tags = ["!dev"];
+VerticalCompact.args = {
+	compact: true,
+	vertical: true
+};
+
+export const HorizontalSizing = (args, context) => Sizes({
+	Template: ActionGroups,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+HorizontalSizing.args = {};
+HorizontalSizing.tags = ["!dev"];
+HorizontalSizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const VerticalSizing = (args, context) => Sizes({
+	Template: ActionGroups,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+VerticalSizing.args = {
+	vertical: true
+};
+VerticalSizing.tags = ["!dev"];
+VerticalSizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When an action group is justified, it takes up the entire available container width, divided equally for each action button that is inside the group.
+*/
+
+export const Justified = ActionGroups.bind({});
+Justified.tags = ["!dev"];
+Justified.args = {
+	customStyles: {"width": "300px"},
+	justified: true,
+	content: [
+		{
+			iconName: "AlignTop",
+			label: "Align Top",
+		},
+		{
+			iconName: "AlignBottom",
+			label: "Align Bottom",
+		},
+	]
+};
+
+export const JustifiedIconOnly = ActionGroups.bind({});
+JustifiedIconOnly.tags = ["!dev"];
+JustifiedIconOnly.args = {
+	customStyles: {"width": "300px"},
+	justified: true,
 	content: [
 		{
-			iconName: "Edit",
-			iconSet: "workflow",
-			label: "Edit",
+			iconName: "AlignTop",
+			label: "",
 		},
 		{
-			iconName: "Copy",
-			iconSet: "workflow",
-			label: "Copy",
+			iconName: "AlignBottom",
+			label: "",
 		},
 		{
-			iconName: "Delete",
-			iconSet: "workflow",
-			label: "Delete",
-			isSelected: true,
+			iconName: "AlignMiddle",
+			label: "",
 		},
-	],
+	]
 };
 
+JustifiedIconOnly.storyName = "Justified (icon-only)";
+
+export const JustifiedIconOnlyCompact = ActionGroups.bind({});
+JustifiedIconOnlyCompact.tags = ["!dev"];
+JustifiedIconOnlyCompact.args = {
+	customStyles: {"width": "300px"},
+	justified: true,
+	compact: true,
+	content: [
+		{
+			iconName: "AlignTop",
+			label: "",
+		},
+		{
+			iconName: "AlignBottom",
+			label: "",
+		},
+		{
+			iconName: "AlignMiddle",
+			label: "",
+		},
+	]
+};
+
+JustifiedIconOnlyCompact.storyName = "Justified (compact, icon-only)";
+
+
+/**
+ * When space is limited in an action group, there are 2 options for the group's overflow behavior: wrap or collapse. By default, an action group is set to wrap, meaning that the action buttons inside the group wrap to form another line. Alternatively, an action group can be set to collapse inside a More (...) action button.
```

Comment:
Do you think we need backticks or something for the 'More (...)' part? I don't think it'd be recommended to use quotes, but maybe backticks would draw attention to that. Or we could wrap it in `**`/a `<strong>` tag like it is on the guidance site? Any thoughts on that?